repos/iiif-storage/src/Controller/PresleyController.php

```diff
@@ -0,0 +1,168 @@
+<?php
+
+
+namespace IIIFStorage\Controller;
+
+
+use Digirati\OmekaShared\Framework\AbstractPsr7ActionController;
+use Digirati\OmekaShared\Model\FieldValue;
+use Digirati\OmekaShared\Model\ItemRequest;
+use IIIFStorage\JsonBuilder\ManifestBuilder;
+use IIIFStorage\Repository\CanvasRepository;
+use IIIFStorage\Repository\CollectionRepository;
+use IIIFStorage\Repository\ManifestRepository;
+use Omeka\Permissions\Acl;
+use Zend\Diactoros\Response\JsonResponse;
+
+class PresleyController extends AbstractPsr7ActionController
+{
+
+    /**
+     * @var ManifestBuilder
+     */
+    private $builder;
+    /**
+     * @var ManifestRepository
+     */
+    private $repo;
+    /**
+     * @var CanvasRepository
+     */
+    private $canvasRepo;
+    /**
+     * @var CollectionRepository
+     */
+    private $collectionRepo;
+    /**
+     * @var Acl
+     */
+    private $acl;
+
+    public function __construct(
+        ManifestBuilder $builder,
+        ManifestRepository $repo,
+        CanvasRepository $canvasRepo,
+        CollectionRepository $collectionRepo,
+        Acl $acl
+    ) {
+        $this->builder = $builder;
+        $this->repo = $repo;
+        $this->canvasRepo = $canvasRepo;
+        $this->collectionRepo = $collectionRepo;
+        $this->acl = $acl;
+    }
+
+    /**
+     * Returns root collection with all manifest collections
+     */
+    public function getRootCollectionAction()
+    {
+        $this->allowCors();
+        $manifests = $this->repo->getNonReplacements();
+
+        return new JsonResponse(
+            [
+                "@context" => "http://iiif.io/api/presentation/2/context.json",
+                "@id" => $this->url()->fromRoute(null, [], ['force_canonical' => true], true),
+                "@type" => "sc:Collection",
+                "members" => array_map(function ($manifest) {
+                    $json = $this->builder->build($manifest, false, 0, 1)->getJson();
+                    $json['service'] = [
+                        "highlight" => "info",
+                        "profile" => "row-profile-here",
+                        "values" => [
+                            "",
+                            $json['label'],
+                            "",
+                            "",
+                            "",
+                            ""
```

Comment:
This is a backwards compatible implementation. We'll start with this and then possibly move to using the metadata on the manifest to display in the Sorting Room.

Reply:
... I don't want to know